source/content/drupal-cms.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Drupal CMS on Pantheon
+description: Learn about installing and managing Drupal CMS sites on Pantheon.
+tags: [site, D11]
+permalink: docs/drupal-cms
+contributors: [alexmoreno, stevector]
+contenttype: [doc]
+innav: [true]
+categories: [create]
+cms: [drupal]
+audience: [development]
+product: [dashboard, terminus]
+integration: [--]
+reviewed: "2024-02-10"
+---
+
+# Drupal CMS
+
+Released in January of 2025, Drupal CMS is a relatively new wrapper around Drupal Core.
+Drupal CMS builds on the concept of [Recipes](https://www.drupal.org/docs/extending-drupal/drupal-recipes) to [radically reduce the amount of time and expertise required to stand up common functionality](https://pantheon.io/blog/drupal-cms-innovations).
+In particular, Drupal CMS is aimed at Marketers and Content Editors who commonly use Drupal, but who may not have the technical expertise to install and configure modules themselves.
+
+For Pantheon's developer-centric community, Drupal CMS will be most valuable as a reference point and proving ground for recipes.
```

Comment:
Is this accurate? This seems to be saying that we don't expect Pantheon customers to get much use out of Drupal CMS.

Reply:
I think recipes will be useful for everyone. I was testing AI chatbot the other day, and installing and configuring those modules took me quite a bit of time, and I failed. Using the recipe I had the bot up and running in minutes. 

I'll propose an update on this text

Reply:
@jazzsequence can you re-review? I'd like to get this PR merged